tasks/task1/task1_2.js

```diff
@@ -0,0 +1,13 @@
+const fs = require('fs');
+const csv = require('csvtojson');
+const { pipeline } = require('stream');
+const errorHandler = require('./utils/errorHandler');
+
+const csvFilePath = 'files/node_mentoring_t1_2_input_example.csv';
+const txtFilePath = 'files/data.txt';
+
+pipeline(
+    csv().fromFile(csvFilePath),
```

Comment:
Consider storing your read and write streams in a separate constants.
Think of your pipeline as a huge job runner - the leaner it is the better.